python/src/trezorlib/cli/cardano.py

```diff
@@ -60,7 +60,7 @@ def sign_tx(client, file, signing_mode, protocol_magic, network_id, testnet):
         protocol_magic = cardano.PROTOCOL_MAGICS["testnet"]
         network_id = cardano.NETWORK_IDS["testnet"]
 
-    signing_mode = messages.CardanoTxSigningMode.__members__[signing_mode]
+    signing_mode = messages.CardanoTxSigningMode[signing_mode]
```

Comment:
no, i mean, this whole line is superfluous because already `isinstance(signing_mode, messages.CardanoTxSigningMode)`

Reply:
Yeah, I just noticed and quickly force pushed the commit 🤦 Sorry.